backends-velox/src/main/scala/io/glutenproject/backendsapi/velox/VeloxBackend.scala

```diff
@@ -307,4 +350,6 @@ object BackendSettings extends BackendSettingsApi {
   override def staticPartitionWriteOnly(): Boolean = true
 
   override def allowDecimalArithmetic: Boolean = SQLConf.get.decimalOperationsAllowPrecisionLoss
+
+  override def requiredRootPaths(): Boolean = true
```

Comment:
Do we need this variable, and in what case the root paths are not required?

Reply:
> Do we need this variable, and in what case the root paths are not required?

I'm not familiar with clickhouse backend so I kept this and enabled it only for velox backend.

Reply:
@zzcclp Could you help check if it's needed for CH backend? Thanks.